### PR TITLE
fix(FEC-12679): ISO Code for Chinese and Spanish in Multi Audio

### PR DESCRIPTION
**Pull request by Tzipi-kaltura**
### Description of the Changes

**the issue:**
when the language track is 639-2 iso code and it's not the 639-1 + additional latter.
(for example: Spanish, 639-1: "es" 639-2: "spa")
the player doesn't recognize the language and it doesn't do the match between codes.

**the solution:**
add fields of extensionAudioLanguage and extensionTextLanguage to the configuration
in order to add additional checking in case the 639-2 iso code is totally different from the 639-1 iso code.

solves FEC-12679

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: flow-typed/types/playback-config.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 declare type PKPlaybackConfigObject = {
   audioLanguage: string,
   textLanguage: string,
+  extensionAudioLanguage: string,
```
Comment (SivanA-Kaltura) [inline flow-typed/types/playback-config.js:5]: I would use "secondaryXLanguage" or "additionalXLanguage"
"extensionXLanguage" is not clear enough

File: src/track/track.js [hunk 2]
```diff
@@ -9,15 +9,21 @@ export default class Track {
...
+  static langComparer(inputLang: string, trackLang: string, extensionLanguage: ?string, equal: ?boolean): boolean {
     try {
       inputLang = inputLang.toLowerCase();
       trackLang = trackLang.toLowerCase();
+      extensionLanguage ? (extensionLanguage = extensionLanguage.toLowerCase()) : extensionLanguage;
       if (equal) {
-        return inputLang ? inputLang === trackLang : false;
+        if (inputLang && inputLang === trackLang) {
+          return true;
+        } else if (extensionLanguage && extensionLanguage === trackLang) {
```
Comment (SivanA-Kaltura) [inline src/track/track.js:24]: why is the && needed ? 
if it's undefined it won't be equal to trackLang

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Comment by SivanA-Kaltura** [comment]
@Tzipi-kaltura note the failing unit test in track.spec.js
you should add more unit tests for the new behavior as well

File: flow-typed/types/playback-config.js (hunk 1 above)
Comment (Tzipi-kaltura) [reply flow-typed/types/playback-config.js]: done

**Review by giladna — APPROVED** [review]

**Pull request merged**